docs/tutorials/integrations/nginx.md

```diff
@@ -0,0 +1,374 @@
+
+This service defines three volumes: the NGINX config file, a self-signed
+certificate, and a private key related to the certificate. Explanation of those
+files follows below. The files are made accessible from the local file system
+for read-only access by the container.
+
+#### Production tips
+
+For the convenience of a tutorial, we automatically generate a self-signed
+certificate and provide it to the proxy service. For reasons that are described
+in more detail [below](#tlstlsconf), this is unsuitable for production Conjur
+deployments.
+
+You can use your own certificate here by providing it to the container as a
+volume. This allows your clients to verify that they are talking to the
+authentic Conjur server. Your security team can provide certificates for your
+organization, or you can create a certificate for any domain or sub-domain you
+control with [certbot][certbot], which uses [Let's Encrypt][lets-encrypt] to
+provide certificates for no cost.
+
+To avoid conflicting with other services that might be running on the tutorial
+user's port 443, we remap the port to 8443. On the production machine, the port
+mapping should be changed to "443:443" instead of "8443:443".
+
+
+[certbot]: https://certbot.eff.org/
+[lets-encrypt]: https://letsencrypt.org/
+
+---
+
+```yaml
+client:
+  image: conjurinc/cli5
+  depends_on: [ proxy ]
+  entrypoint: sleep
+  command: infinity
+```
+
+This service uses the `cli5` image with Conjur CLI pre-installed for convenient
+tinkering. It is connected to the proxy service, allowing it to access Conjur
+via TLS.
+
+The "sleep" and "infinity" bits ensure that this container stays up for the
+duration of the demo. Without these options, the `conjurinc/cli5` image gives
+you an ephemeral stateless client container that performs a single command and
+exits, a desirable behavior for common ops use cases.
+
+[postgres-image]: https://hub.docker.com/_/postgres/
+[nginx-image]: https://hub.docker.com/_/nginx/
+
+### default.conf
+
+This configuration file tells NGINX how to use TLS and behave as a proxy for
+Conjur.
+
+```nginx
+listen              443 ssl;
+server_name         proxy;
+access_log          /var/log/nginx/access.log;
+```
+
+This block sets up a few basic properties of the NGINX server. Its hostname is
+`proxy`, it listens on the standard port for HTTPS (port 443) and it has a
+location for its access logs, useful for monitoring traffic.
+
+---
+
+```nginx
+ssl_certificate     /etc/nginx/nginx.crt;
+ssl_certificate_key /etc/nginx/nginx.key;
+```
+
+This block gives NGINX its directions on how to perform TLS ("ssl" is a name
+for an older standard for TLS and is still often used interchangeably.)
+
+The `certificate` is a public key, and the `certificate_key` is the
+corresponding private key. NGINX maintains [documentation][nginx-https] about
+how to configure the server for HTTPS, including production optimization
+guidelines and the values of many default settings, so that is worth reading.
```

Comment:
I'd kill ", so that is worth reading" and maybe just add "useful" before the documentation link

Reply:
changed